content/python/concepts/errors/errors.md

```diff
@@ -79,6 +79,17 @@ import notamodule
 ModuleNotFoundError: No module named 'notamodule'
 ```
 
+### Type Error
+
+`TypeError` is thrown when a function's argument is of an inappropriate type.
+
+```error
```

Comment:
```suggestion
```shell
```

Since \```error [doesn't seem to be supported](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/#code-blocks), and this is technically output, can we replace all instances of \```error with \```shell?